src/ankit/arrayConcepts/SimpleCalculation.java

```diff
@@ -0,0 +1,70 @@
+package ankit;
+
+public class SimpleCalculation {
+	int index = 0;
+
+	int minimumNumber(int[] arr) {
+		int minimum = arr[0];
+		for (index = 0; index < arr.length; index++) {
+			if (minimum > arr[index])
+				minimum = arr[index];
+		}
+		System.out.println("From the list of array provided by client, minimum number is : " +minimum);
+		return minimum;
+	}
+
+	int maxiumNumber(int[] arr) {
+		int maximum = arr[0];
+		for (index = 0; index < arr.length; index++) {
+			if (maximum < arr[index])
+				maximum = arr[index];
+		}
+		System.out.println("From the list of array provided by client, maximum number is : " +maximum);
+		return maximum;		
+	}
+
+	int avgerageCalculation(int[] arr) {
+		int avg = 0;
+		int sum = 0;
+		int counter = 0;
+		for (index = 0; index < arr.length; index++) {
+			sum += arr[index];
+			counter++;
+		}
+		avg = sum / counter;
+		System.out.println("Average of the list of array provided is : " +avg);
+		return avg;
+	}
+
+	void displayStraight(int[] arr) {
+		for(index=0; index<arr.length; index++) {
+			System.out.print(arr[index]+ " ");
+		}
+		System.out.println();
+	}
+
+	void displayReverse(int[] arr) {
+		for(index=(arr.length-1); index>=0; index--) {
+			System.out.print(arr[index]+ " ");
+		}
+		System.out.println();
+	}
+	
+	void averageOfMinAndMaxNumbers(int min, int max) {
+		int averageMinMax =0;
```

Comment:
I think we have requirement where you need to find max & min number then you need to find avg of both, so this expecting two variable in method looks wrong as per the implementation perspective, user will give us array, not min & max value. I hope you got my point.


You can write code like this, I remove variable and directly doing sum of min & max, no problem, if you are not comfortable to write code like below, just optimize.

int avgOfMinMax = (getMinNumber(input)+getMaxNumber(input))/2